great_expectations_cloud/agent/agent.py

```diff
@@ -205,11 +216,12 @@ def _listen(self) -> None:
             AuthenticationError,
             ProbableAuthenticationError,
             AMQPConnectorException,
+            AMQPConnectionError,
```

Comment:
This appears to have been the main issue; the error we saw was a `ConnectionClosedByBroker`, which extends `AMQPConnectionError`.